content/docs/demos/prometheus_grafana.md

```diff
@@ -1,49 +1,61 @@
 ---
-title: "Prometheus & Grafana"
-description: "Describes basic steps to get up and running a Prometheus and Grafana stack with OSM specific configuration and Dashboards"
+title: "Integrate OSM with Prometheus and Grafana"
+description: "Describes how to set up an OSM-specific configuration and dashboards with your own Prometheus and Grafana stack"
 type: docs
 weight: 25
 ---
 
```

Comment:
I think all other demos have a list of prerequisites at the beginning of the article. Could we add this here?

Reply:
fixed

Reply:
fixed